nightly/Dockerfile

```diff
@@ -8,14 +8,16 @@ LABEL maintainer="Citus Data https://citusdata.com" \
       org.label-schema.version="Nightly" \
       org.label-schema.schema-version="1.0"
 
-# switch to Citus nightly
+# Perform prerequisites for Citus Nightly Installation
 RUN apt-key del 1530DF18 \
     && rm -rf /etc/apt/sources.list.d/citusdata_community.list \
     && apt-get update \
     && apt-get install -y --no-install-recommends \
        ca-certificates \
-       curl \
-    && curl -s https://install.citusdata.com/community-nightlies/deb.sh | bash \
+       curl
+
+# Install Citus nightly
+RUN curl -s https://install.citusdata.com/community-nightlies/deb.sh | bash \
     && apt-get install -y postgresql-$PG_MAJOR-citus \
     && apt-get purge -y --auto-remove curl \
-    && rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/* \
```

Comment:
Why do you need `\` at the end? I guess this is the last line and it is not needed.

Reply:
Fixed